spec/models/miq_server/role_management_spec.rb

```diff
@@ -1,4 +1,4 @@
-describe "Server Role Management" do
+RSpec.describe "Server Role Management" do
   context "After Setup," do
     before do
       @csv = <<-CSV.gsub(/^\s+/, "")
```

Comment:
Think you can remove this as well.

Reply:
In rspec 4 the outer describe block will require `RSpec.describe`, so it's deliberate.

Reply:
What? I'm talking about the `@csv` variable ... it isn't used now that you've removed the stub.

Reply:
Ah, right. Updated. :)